UnityProjects/MRTKDevTemplate/Assets/Plugins/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just to double check, all the build preprocessors that add manifest entries based on OpenXR features still work with a custom manifest, right?

Reply:
I don't have access to a Quest right now, so could @Zee2 or @keveleigh help me to verify this?

Reply:
Also I actually don't know how to tell even if I had a device right now.

Reply:
I can give it a test! The most obvious one would be hand tracking access

Reply:
Thanks! Will be waiting for your results. Please test it with the input field scene to ensure the custom manifest is not being overwritten as well.

Reply:
Seems good, including keyboard!